debian/upstream/signing-key.asc

```diff
@@ -29,48 +24,29 @@ Vtvua2wFGj9DrxO+WQIPZrX/jJu7ntqxL2I4zLKRrvnMjAGtjT3I7zLsMvLaZN02
-7L3JGDlPqKmG4qStrgzSxb/8BpaJAhwEEAEIAAYFAlcVTJQACgkQRo8SLOgWIpXu
-BA/+N5D+u+0HHaWLXN7hYEVgASwCnmBzCVaJyraztO2thDOdiKiUHSxa4bj8gmaZ
-/MsFGTG/h0ToFDU4WqeFs83NIVONF0qAbKbwOc+TRS5uihKrft+tfB412cFYNlds
-SLvWc3EIAO15RCMSdiK0i/L8vQr8E7Bg8U+BCpCQBM/nIzkpazqXzCoFaj77M94j
-aZuLz11DdWMKWW2/Scv9cIWILFkn0vamn8v0QIfqnLRk6vTgGsMnBsYJSm+WUMek
-ZenMu9FyDvLZ7iKrwyVyyOTfdHxYoU7AebIKX6AlgqmL5dK1hox8QdedfppNviJQ
-d0Wisrryg+0Y0GcWdAGqYOvTpEYv05gNlICm7WeQBX6DWdxGHJ7637iLzRGF0bzB
-s3HilDDm/2IVe9zTW3+U683xZgk6GCDEYI6P1D8oPAj2zcrJDvOdJXNL1rS7f0LG
-tgngypJPUjOtePCtUyZbo5VUUBHapJzMGG4jssXwHJJA3Bajjhs2LHG/lQ6pqoUF
-hKyZX5/tCtC1hGXyGzoUM7kx4J8NvKKQGdNpfYjLcIvfn8KunudWMZKsKFJlsNWH
-v/hYnOfT0K9Eb7VUqeUcy4f08hd3CEIS3mAquqRqlUtx+jlJ90K5c7khGJECo3Tt
-S6GqTRVcvTZ4Lzd/6md7+WTwn8e6VQTiZQ1sA0yO15ab5LmJATMEEAEIAB0WIQST
-bAUBkV9m7+JTK/ANiLCEj91ImwUCWV2ksAAKCRANiLCEj91Im6leB/4vq7BCnG8x
-hSwjGhNGsyFITXCgazIs0rGvgZsovKVmytoJe0nlcW2RkHQ1z7Kl8AM00mWSXqeN
-yp50eF7sQ/Q4lCH/NCOkGQ/qG15a+ksynBIHkd4KVydoWDW+3pikqYmpf7Ba+eVc
-jwhmuzYwvUTHSZJtAJYpO/yuSbWUYRRF37VQZfogr7E5fnI8g8d8VF/9/akS/BN/
-z7gVQZ7glVgrV9amBxToPopMu+A1Yet1udT0vRzln7LadlqB8YLYSETS4MX4IVds
-WrgNjegnVojUsaZUE29XpzThdrYArf1PPKG3aOFPZDtes7NqQpXemUR6QPkfbG9o
-QbgYUEt/4/c5iEYEEBECAAYFAlwRLbcACgkQSTYLOx37oWTrOQCePmNbdCgEFozp
-SpfYkgeoMIRkchIAoI5kzXMvkpIueiUHLAaQJ2jrq1YruQINBFb+zdwBEAC8XHJe
-AKacMAOLQnMSnQtB6EuUfxG6dwBRqKyBQi0TKGs8ix/eXx4KO7JbCu+9On3f6BGx
-4W9UFytz8AoK+onCuKjJOulbDB9pHxyBMFmyktIX5HAZTgoJY4Pq+A2gzzz97fWj
-snnWHAnrs+DbnHHaw5kLQVz2UOxPT3nL+2AsLUGEqd+NfamelUfnw1iGv0U+cJjj
-h87z/GUeSfYY7dc81/727PU2Hl+hX6+EwKnnUhyxLLRz8CVeBdSDO7lnWYiqtqAU
-9+r+C2q6qbFWkSMXqYxlmCmDPIDwZhc92CzAitYurfkYkP4s0H4XwQ7QT2ibZ782
-P7Oo6MlAt282XogNCeo1KDBdXyQZfsYW8ZzydWRFUurMhIn0ZIWk31zhBH9T2VVD
-x6E3n8y5wTzKZkUzx3fFQ0ZpdQTHlYAFODhIY1bi++UIhaf/IlrB+qDcDhuIZvcy
-quJziHKLAYv65yUkoHv3vyqHnAR2A5tkJ3oPyAlQ8FfL691eE1r067u/8yVw9VzM
-J2wd7oKF8gqC+6+BNcssLv3xIqcEeRRBLgQNK2/c0m7g66DIKT1BzyI6ZfdhbxiZ
-1NLvWKz3u28eNYS/2ACbIcxCzytgtLztXPvy0D3/clmgalqiw3udYxJxtePSjEa2
-D213OGdw8B/CyiYwW+Dg5/MlYHY+liXoAjgN2wARAQABiQI8BBgBCgAmAhsMFiEE
-i6abh9Q74pTyPoEgiaKtPAfZYugFAlyTw8sFCQlXav8ACgkQiaKtPAfZYuh0Zw//
-ZUtqiOv2EEQPlL8KyFJ+DC/KGOFyEmcwvql88P6njF+E+0JQr1ItuDKZX12Y0hfo
-yr5HK8/ZTZB/rrOAHBLv3+4/EHQF8Sl+qs/ab7cxIXqWEZOeNVX31EWfgIqUFf+6
-BBazqxOlZQgR6U8ik9XG43Fonfu6oYPjq4sC7VM2sL5nzRb4s5BqHqC/AjRq7zne
-NH9ssMCjpaZtA79hUHgMtYvoExZW3IUyZyQcKx1riY01LtQ5gYQz1bkEangvXZzR
-o06wYcZ4h1rbeIk37eOZJ+pyM1FicDD1OExwO0oTLoaUIJYJdJSqHegyt6Gh57mA
-G3C4CogmNe13vjpzW5e6/jpcq9fvjMd7iW+kV8+IrEYcw8xYo4YB7MiZYw9EGkiN
-cg22w2wkqJUDQQSIFI6oUJE8iL2e1twZXfjcoA3uFsaSP0823DTFKwkjNAZVTfWT
-tZzLzsmMVjRjJd72VvN9ChZYlQZCkkBMNQuisIAG9/kLp7KQxELClI7gB1/yJGdJ
-dSZBzF0b2Na+3co13ZJCKpnvEg7AOH+Dys1cV3FO45/kSmkBD+cUAdJeD7zMpTRM
-jhk8lyMNB4JtWXMfW8V39M2IkG+YH0d6ogaqAjNv5/IyR/whpkzcRlgqRUdvEtcB
-2Oemsf7tTE2CD/bGRp2Byezdyl7q2JaUXY8WXUQ05ko=
-=2tbg
------END PGP PUBLIC KEY BLOCK-----
\ No newline at end of file
+7L3JGDlPqKmG4qStrgzSxb/8Bpa5Ag0EVv7N3AEQALxccl4AppwwA4tCcxKdC0Ho
```

Comment:
Just curious, why is your pubkey block shorter now?

Reply:
I exported with the `export-minimal` option ([as suggested by Lintian](https://lintian.debian.org/tags/public-upstream-key-not-minimal.html)). 

From the GPG manpages:

```
export-minimal
  Export the smallest key possible. This removes all signatures except the most
  recent self-signature on each user ID. This option is the  same as running
  the --edit-key command "minimize" before export except that the local copy of
  the key is not modified. Defaults to no.
```